k8s/resources/airflow_basic_auth_create_user.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
Document in readme?

Reply:
I added something to the k8s README - do you mean something over and above that?